src/printapp/templates/index.html

```diff
@@ -292,6 +292,10 @@ <h3 class="subheader">Welcome, {{email}}</h3>
         <strong>Incorrect file type.</strong> Sorry, we can’t print files of this type. Try a Word document or PDF.
         <a href="#" class="close">&times;</a>
       </div>
+      <div id="fileUploadError" data-alert hidden class="alert-box alert radius">
+        <strong>Sorry!</strong> Something went wrong while uploading your file. Please try again.
+        <a href="#" class="close">&times;</a>
+      </div>
```

Comment:
If you upload another file, does this go away? Or is it a non-issue because you're forced to refresh the page.

It's not a big deal, but consider hiding the alert when starting a new upload.